torch/cuda/nvtx.py

```diff
@@ -0,0 +1,48 @@
+import ctypes
+
+lib = None
+
+__all__ = ['range_push', 'range_pop', 'mark']
+
+
+def _libnvToolsExt():
+    global lib
+    if lib is None:
+        lib = ctypes.cdll.LoadLibrary(None)
+        lib.nvtxMarkA.restype = None
```

Comment:
Nope, they return integers describing how many levels of nesting the ranges have.